SwiftUI-Xcode-Template/SwiftUI-Xcode-Template.xcodeproj/project.pbxproj

```diff
@@ -470,15 +470,14 @@
 			};
 			name = Release;
 		};
-		3F6C39D02C3A9A1600776C37 /* Debug */ = {
+		3FDF49EA2C3A9CAF00CFCFDE /* Debug */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
-				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES;
```

Comment:
Interesting... By removing this, it fallbacks to the iOS Default `NO`:

<img width="1764" alt="image" src="https://github.com/mokagio/SwiftUI-Xcode-Template/assets/1218433/7ae408db-7f1d-4ba1-a105-dbd0d69ab83a">